src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml

```diff
@@ -9380,7 +9380,7 @@
       </Setter.Value>
     </Setter>
   </Style>
-  <Style TargetType="GridViewItem" BasedOn="{StaticResource DefaultGridViewItemStyle}" />
```

Comment:
Should we have UNO TODO comment here is well?

Reply:
It seems the XAML merge actually omits top-level comments from the merged file, so that's why both of them are missing here. As the file is generated, it should be a problem.

Reply:
Oh I wasn't aware this file is generated.